src/styles/stars.css

```
@@ -0,0 +1,26 @@
+.card__stars {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.card__star {
+  width: 16px;
+  height: 16px;
+
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__star:not(:nth-child(5n)) {
```

looks easier this way
```suggestion
.card__star:not(:last-child) {
```